doc/classes/HTTPRequest.xml

```diff
@@ -69,16 +69,19 @@
 			<argument index="0" name="path" type="String">
 			</argument>
 			<description>
-				Set the file to download into. Outputs the response body into the file.
+				Sets the file to download into. Outputs the response body into the file if set.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="body_size_limit" type="int" setter="set_body_size_limit" getter="get_body_size_limit">
+			Maximum size of response bodies that should be accepted.
```

Comment:
In general, it's better to avoid prepositional phrases in favor of direct statements:

Maximum allowed size for response bodies.